src/sandboxes/impl/seatbelt.py

```
@@ -0,0 +1,213 @@
+        )
+        path = _write_temp_profile(content)
+        with self._lock:
+            if for_mcp:
+                self._mcp_profiles.append(path)
+            else:
+                # Clean up previous tool profile to prevent temp file leaks
+                if self._tool_profile:
+                    try:
+                        os.unlink(self._tool_profile)
+                    except Exception:
+                        pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)